configure.ac

```diff
@@ -0,0 +1,115 @@
+# IBM_PROLOG_BEGIN_TAG
+# This is an automatically generated prolog.
+#
+# $Source: configure.ac $
+#
+# OpenPOWER sb-signing-utils Project
+#
+# Contributors Listed Below - COPYRIGHT 2016
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# IBM_PROLOG_END_TAG
+
+AC_INIT(SecurebootSigningUtils, 1.0.0, subras@us.ibm.com)
+AC_CONFIG_AUX_DIR([config])
+AM_INIT_AUTOMAKE([-Wall -Werror foreign -Wno-portability])
+AM_SILENT_RULES([yes])
+AC_CONFIG_MACRO_DIR([m4])
+
+# not using config.h yet
+AC_CONFIG_HEADER([config.h])
+
+# Checks for programs.
+AC_PROG_CC
+AC_PROG_RANLIB
+AC_PROG_CXX
+
+# Checks for c++11 compatibility
+AX_CXX_COMPILE_STDCXX_11([noext],[optional])
```

Comment:
I know that IBM has some truly ancient build machines (RHEL6). Does this work on RHEL6?